import java.io.IOException;

class Testthrows4 {
	
	public void nams()throws IOException{
		System.out.println("dfkjndf");
	}
	
	public static void main(String args[]) {// declare
																// exception
		Example8 m = new Example8();

		m.method();
		
		System.out.println("normal flow...");
	}
}